k8s/eks/README.md

```diff
@@ -0,0 +1,282 @@
+This guide will help you setup an EKS (Elastic Kubernetes Service) cluster, and run testground plans on it.
+
+## Requirements
+
+It is assumed that you already have the following:
+
+- A bastion host on AWS, or a laptop with AWS credentials which will be used to run the installation script, interact with the cluster, and run testground plans
+- The laptop or bastion host has the following software installed:
+
+**Note: the software and installation guides have been tested on amazon linux. Please verify which OS are you using and follow the official guides accordingly.
+Links have been included for each software requirement. Listed guides are for example purposes only.**
+
+**1. docker**
+
+Official installation guide:
+
+https://docs.docker.com/engine/install/ubuntu/
+
+Installation on amazon linux:
+
+```
+sudo yum install docker -y
+sudo systemctl enable docker.service
+sudo systemctl start docker.service
+sudo usermod -aG docker ec2-user
+newgrp docker
+```
+
+**2. helm**
+
+Official installation guide:
+
+https://helm.sh/docs/intro/install/
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+**3. kubectl**
+
+Official installation guide:
+
+https://kubernetes.io/docs/tasks/tools/
+
+Latest version:
+
+```
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+kubectl version --client
+```
+
+Specific version (required when your cluster is not compatible with the latest version):
+
+```
+curl -LO https://dl.k8s.io/release/v1.22.0/bin/linux/amd64/kubectl
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+kubectl version --client
+```
+
+**4. AWS CLI (v2) with AWS credentials and ECR login**
```

Comment:
Do we use ECR login during the installation script?

Reply:
We do not, that's why this is in the README:
```
It is assumed that you already have the following:
- A bastion host on AWS, or a laptop with AWS credentials which will be used to run the installation script```

Reply:
But we can remove `and ECR login` part, right?

Reply:
Well when it comes to the bastion, I believe we can remove it. I need to get back to you on this one, need to test if it is safe to remove.

Reply:
ECR login part has been removed from the README.